src/nodes/common/Custom.ts

```diff
@@ -22,18 +22,18 @@ export default class Custom extends Node {
   }
 
   public load(state: INodeState) {
+    this.updateNode(state);
     super.load(state);
-    this.updateNode();
```

Comment:
Can you explain what has changed now? Surely when calling `super.load(state)` that sets the state which means it will be available for `this.updateNode()`

Reply:
Yes, but `super.load(state)` also attempts to set the connections, which couldn't succeed because the interfaces do not exist yet!

Reply:
Ah I see, thanks for that